xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
+     &exampleuser; (usually <filename>/home/&exampleuser_plain;</filename>).
+    </para>
+   <para>
+    To copy a directory remotely is similar to copy a directory locally.
+    The following example copies the directory <filename>&exampleuser_plain;/</filename> and
+    its content into the remote directory <filename>/var/backup/</filename> on
+    the <systemitem class="ipaddress">&exampledomain1ip;.1</systemitem> host:
+   </para>
+   <screen>&prompt.user;<command>rsync</command> -avz &exampleuser_plain; &exampledomain1ip;.1:/var/backup/</screen>
+   <para>
+    Assuming you have write permissions on the host <systemitem class="ipaddress">&exampledomain1ip;.1</systemitem>, you will find the
+    copy in the absolute path <filename>/var/backup/&exampleuser_plain;</filename>.
    </para>
-   <para>
-    In this case, look at the conflict marks
-    (<quote><literal>&gt;&gt;</literal></quote> and
-    <quote><literal>&lt;&lt;</literal></quote>) in the file and decide between
-    the two versions. As this can be a rather unpleasant job, you might decide
-    to abandon your changes, delete the local file, and enter
-    <command>cvs</command>&nbsp;<option>up</option> to retrieve the current
-    version from the server.
-   </para><indexterm class="endofrange" startref="idx.CVS"/><indexterm class="endofrange" startref="idx.files_synchronizing_CVS"/>
   </sect2>
- </sect1>
- <sect1 xml:id="sec.net.sync.rsync">
-  <title>Introduction to rsync</title><indexterm>
 
-  <primary>rsync</primary></indexterm>
-
-  <para>
-   rsync is useful when large amounts of data need to be transmitted regularly
-   while not changing too much. This is, for example, often the case when
-   creating backups. Another application concerns staging servers. These are
-   servers that store complete directory trees of Web servers that are
-   regularly mirrored onto a Web server in a DMZ.
-  </para>
-
-  <sect2 xml:id="sec.net.sync.rsync.usage">
-   <title>Configuration and Operation</title>
-   <para>
-    rsync can be operated in two different modes. It can be used to archive or
-    copy data. To accomplish this, only a remote shell, like SSH, is required
-    on the target system. However, rsync can also be used as a daemon to
-    provide directories to the network.
+  <sect2 xml:id="sec.net.sync.rsync.server">
+   <title>Configuring and Using a Rsync Server</title>
+   <para> Rsync can be operated as a daemon (<systemitem class="daemon"
+     >rsyncd</systemitem>) listing on default port 873 for incoming
+    connections. This daemon can receive <quote>copying targets</quote>.
    </para>
    <para>
-    The basic mode of operation of rsync does not require any special
-    configuration. rsync directly allows mirroring complete directories onto
-    another system. As an example, the following command creates a backup of
-    the home directory of &exampleuser; on a backup server named
-    &exampleserver;:
+    The following procedure creates a Rsync server on
```

Comment:
The following procedure creates a Rsync server

**The following description explains how to create an Rsync server**